adam-pipeline/wrapper.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import os
+import shutil
+import subprocess
+import json
+import logging
+from uuid import uuid4
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger()
+
+
+def write_config(mount, args):
+    
+    # open path to write config file at
+    path = "adam-preprocessing-%s.config" % (str(uuid4()))
+    fp = open(path, 'w')
+
+    # generate config
+    config = '''# ADAM preprocessing pipeline configuration file; autogenerated, do not modify!
```

Comment:
Consider textwrap.dedent which is part of Python and lets you indent the body of the multi-line string.